bob/src/cluster/quorum.rs

```diff
@@ -318,6 +319,91 @@ impl Quorum {
         }
         Ok(())
     }
+
+    async fn collect_remote_exists(
+        result: &mut [bool],
+        keys: &[BobKey],
+        indexes_by_node: &mut HashMap<Node, IndexMap>,
+    ) -> Result<(), Error> {
+        if !indexes_by_node.is_empty() {
+            let mut keys_by_node = HashMap::new();
+            for (node, node_map) in indexes_by_node.iter_mut() {
+                node_map.retain_not_existed(&result);
+                if !node_map.is_empty() {
+                    keys_by_node.insert(node.clone(), node_map.collect(keys));
+                }
+            }
+
+            if !keys_by_node.is_empty() {
+                let nodes: Vec<_> = keys_by_node.keys().collect();
+                let remote_results = exist_on_remote_nodes(&nodes, &keys_by_node).await;
+                for (remote_result, node) in remote_results.into_iter().zip(nodes) {
+                    match remote_result {
+                        Ok(remote_result) => {
+                            debug_assert!(node.name() == remote_result.node_name());
+                            indexes_by_node
+                                .get(node)
+                                .expect("node should exist")
+                                .update_existence(result, remote_result.inner());
+                            trace!("Check existence on node {}: found {} keys", 
```

Comment:
Add total number of keys to output, that was sent to the node: `found {}/{} keys`